rdr_service/genomic/genomic_job_controller.py

```diff
@@ -297,41 +304,116 @@ def ingest_member_ids_from_aw1_raw_table(self, member_ids):
 
                 for r in update_recs:
 
-                    # Set RDR attributes: aw1_file_processed_id, job_run_id, gc_site
-                    self.set_rdr_aw1_attributes_from_raw(r, file_proc_map)
+                    # AW1
+                    if n == 1:
+                        self.set_rdr_aw1_attributes_from_raw(r, file_proc_map)
 
-                    # Set attributes
-                    member = self.set_aw1_attributes_from_raw(r)
+                        self.set_aw1_attributes_from_raw(r)
 
-                    s.merge(member)
-                    completed.append(member.id)
+                    # AW2
+                    else:
+                        self.preprocess_aw2_attributes_from_raw(r, file_proc_map)
 
-        # Output results
-        result_msg = ''
-        result_msg += 'Ingestion From Raw Results:'
-        result_msg += f'    Updated IDs: {completed}'
-        result_msg += f'    Missing IDs: {missing}'
-        result_msg += f'    Multipls found for IDs: {multiples}'
+                        metrics_obj = self.set_validation_metrics_from_raw(r)
 
-        return result_msg
+                        metrics_obj = s.merge(metrics_obj)
+                        s.commit()
+                        metrics.append(metrics_obj.id)
+
+                    s.merge(r[0])
+                    completed_members.append(r[0].id)
+
+            # BQ Updates
+            if n == 2:
+                # Metrics
+                bq_genomic_gc_validation_metrics_batch_update(metrics, project_id=self.bq_project_id)
+                genomic_gc_validation_metrics_batch_update(metrics)
+
+            # Members
+            bq_genomic_set_member_batch_update(metrics, project_id=self.bq_project_id)
```

Comment:
I'm wondering how big these batches end up being.   We may want to talk (with Robert to make sure I'm not suggesting something ill-advised?) about whether we can try to make use of the resource-rebuild task queues to farm out these requests to rebuild the data for bigquery/resource generators.   

This would specifically be for performance improvements, but it could make the local tool runs much less costly.

Reply:
Closing the loop on this, we'll want to implement the resource-rebuild task queues for this. We can implement this in a future PR.